recipes/pymodsecurity/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pymodsecurity" %}
+{% set version = "0.0.1" %}
+{% set build_number = 0 %}
+{% set hash_value = "e9570494282b71172f909cb3ecfd4b7b66fa29a3141439192a48b40939893495" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: {{ build_number }}
+  skip: true  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - modsecurity
```

Comment:
``modsecurity`` should not be here, your compilation is getting libraries from the system or the build environment, the right way is to get from the ``host`` section the libraries which you need to compile.

I think @synapticarbors also tried to tell you that

Reply:
Now I understood why it's getting the libraries from build host, you put

https://github.com/actions-security/pymodsecurity/blob/f9663cfa8341a6037204aee8d812bdc8f2cf7a9b/CMakeLists.txt#L10

in your ``CMakeList``, to be more generic you can just remove it and pass as an option to your ``cmake`` the right options to see the libraries like
```
cmake . -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX -DCMAKE_PREFIX_PATH:PATH=$PREFIX
```

Instead of installing that using the ``setup.py`` you can just compile it and copy the shared object to the ``site-packages``

Reply:
But I think to do that you have to get from tarballs instead of your pypi package

Otherwise you have to pass to the ``setup.py`` the right location of the ``host`` libraries

Reply:
It's also getting the ``build`` libraries using just the ``setup.py`` to compile because of
https://github.com/actions-security/pymodsecurity/blob/f9663cfa8341a6037204aee8d812bdc8f2cf7a9b/setup.py#L26

Reply:
``Conda`` usually is transparent to the installation or at runtime, if you do this kind of strategy you will mess your package

Reply:
Oh I see! Got it now! Sorry for the misunderstanding. We have removed the hardcoded conda prefixes from the code and the wrong dependencies in the `build` section, although we prefer to keep the build process with pip to avoid copying files by hand if that's ok.